lib/rest/api_client.cpp

```diff
@@ -255,25 +257,25 @@ namespace disccord
 
             pplx::task<void> rest_api_client::create_reaction(uint64_t channel_id, uint64_t message_id, std::string emoji, const pplx::cancellation_token& token)
             {
-                auto route = get_route("PUT", "/channels/{channel.id}/messages/{message.id}/reactions/{emoji}/@me", std::to_string(channel_id), std::to_string(message_id), emoji);
+                auto route = get_route("PUT", "/channels/{channel.id}/messages/{message.id}/reactions/"+util::url_encode(emoji)+"/@me", std::to_string(channel_id), std::to_string(message_id));
```

Comment:
This didn't really need to change much, did it? Just wrap the original emoji parameter in `util::url_encode`?